modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -3,30 +3,45 @@
 // * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
 
 [id="nw-ovn-kubernetes-migration-about_{context}"]
-= Migration to the OVN-Kubernetes network plugin
+= Offline migration to the OVN-Kubernetes network plugin overview
 
-Migrating to the OVN-Kubernetes network plugin is a manual process that includes some downtime during which your cluster is unreachable. Although a rollback procedure is provided, the migration is intended to be a one-way process.
+The offline migration method is a manual process that includes some downtime, during which your cluster is unreachable. This method is primarily used for self-managed {product-title} deployments. 
 
-A migration to the OVN-Kubernetes network plugin is supported on the following platforms:
-
-* Bare metal hardware
-* Amazon Web Services (AWS)
-* Google Cloud Platform (GCP)
-* {ibm-cloud-name}
-* Microsoft Azure
-* {rh-openstack-first}
-* VMware vSphere
-* Nutanix
+Although a rollback procedure is provided, the offline migration is intended to be a one-way process.
 
+////
 [IMPORTANT]
 ====
 Migrating to or from the OVN-Kubernetes network plugin is not supported for managed OpenShift cloud services such as {product-dedicated}, Azure Red Hat OpenShift(ARO), and Red Hat OpenShift Service on AWS (ROSA).
 ====
-
+////
 include::snippets/sdn-deprecation-statement.adoc[]
 
+The following sections provide more information about the offline migration method.
+
+[id="supported-platforms-offline-migrating-ovn-kubernetes"]
+== Supported platforms when using the offline migration method
+
+The following table provides information about the supported platforms for the offline migration type.
+
+.Supported platforms for the offline migration method
+[cols="1,1", options="header"]
+|===
+| Platform              | Offline Migration
+
+| Bare metal hardware             |&#10003;         
+| Amazon Web Services (AWS)       |&#10003;         
+| Google Cloud Platform (GCP)     |&#10003;         
+| {ibm-cloud-name}                |&#10003;         
+| Microsoft Azure                 |&#10003;         
+| {rh-openstack-first}            |&#10003;         
+| VMware vSphere                  |&#10003;        
+| AliCloud                        |                 
+| Nutanix                         |                 
```

Comment:
Alicloud and Nutaix are also supported. And we shall also mention both IPI and UPI clusters are supported.